index.html

```diff
@@ -129,8 +192,13 @@ <h3>Element Attributes</h3>
   </style>
 
   <script>
+  const init = async() => {   
     const reader = document.querySelector('m-camera-barcode-scanner');
 
+    let cameraId = await getCameraId('6366'); 
```

Comment:
Just to confirm, is this the device ID of the bottom camera of the Marriott Kiosk device?